src/com/missafternoon/budjen/transactions/CurrencyTest.java

```diff
@@ -1,23 +1,25 @@
+package com.missafternoon.budjen.transactions;
+
 import org.junit.Test;
 
 import static org.junit.Assert.assertEquals;
 
-public class CurrencyTest {
+class CurrencyTest {
```

Comment:
Ooops! These tests won't run anymore.

JUnit 4 classes have to be `public`, and also the `@Test` annotated methods.